.travis.yml

```diff
@@ -4,7 +4,9 @@ dist: trusty
 sudo: required
 node_js: stable
 install:
-  - npm install
-  - bower install --production
+- npm install -g bower pulp purescript@0.11.7
```

Comment:
`npm i` will install this libs too so there should be no need for this.
also in travis, binaries of all installed npm libs is available in PATH